### ISSUE TITLE
Controller Activity Report

### ISSUE THREAD

**Issue by jvoss** [jvoss/artccmgr#3]
In gitlab by @mkosmo on Oct 17, 2017, 03:38

**Comment by jvoss** [comment]
In gitlab by @mkosmo on Oct 17, 2017, 04:29

Depends on #5.

**Comment by jvoss** [comment]
In gitlab by @jvoss on Oct 28, 2017, 20:28

mentioned in commit 6e970208c51e34e4e1b6931936230ac8347b536a

**Comment by jvoss** [comment]
In gitlab by @jvoss on Oct 28, 2017, 20:30

Added controller activity report in 6e970208

Additionally, settings are available to set custom controlling hour requirements for staff, visiting controllers, and local controllers separately. Reports are based on a one month window.

Pending merge into development.

**Comment by jvoss** [comment]
In gitlab by @jvoss on Oct 28, 2017, 20:33

mentioned in merge request !10

**Comment by jvoss** [comment]
In gitlab by @jvoss on Oct 28, 2017, 20:34

mentioned in commit c91ce1e3e700ed46d77f93236f3b6ad3190a290f

**Comment by jvoss** [comment]
In gitlab by @jvoss on Oct 28, 2017, 20:34

Merged into development.

**Comment by jvoss** [comment]
In gitlab by @jvoss on Oct 28, 2017, 20:34

closed
